examples/apple/mps/scripts/mps_example.py

```diff
@@ -64,7 +71,8 @@
     ).to_edge(exir.EdgeCompileConfig(_check_ir_validity=False))
     logging.info(f"Exported graph:\n{edge.exported_program.graph}")
 
-    lowered_module = to_backend(MPSBackend.__name__, edge.exported_program, [])
+    compile_specs = [CompileSpec("use_fp16", bytes([args.use_fp16]))]
```

Comment:
@georgepaw can you add a check here? Basically make it conditional based if it's enabled/disabled from the command line.
```suggestion
    compile_specs = []
    if args.use_fp16:
        compile_specs = [CompileSpec("use_fp16", bytes([args.use_fp16]))]
```